server/src/main/java/io/crate/protocols/postgres/ConnectionProperties.java

```diff
@@ -61,15 +118,8 @@ public Protocol protocol() {
         return protocol;
     }
 
+    @Nullable
     public Certificate clientCert() {
-        // This logic isn't in the constructor to prevent logging in case of SSL without (expected) client-certificate auth
-        if (sslSession != null) {
```

Comment:
Logging is done in `ClientCertAuth`  -  same as before except that we don't have exception there. 
Should I also store the exception  in a dedicated field to fully replicate the previous behaviou?